client/src/app/+admin/overview/users/user-list/user-list.component.ts

```diff
@@ -133,10 +145,12 @@ export class UserListComponent extends RestTable <User> implements OnInit {
     this.saveSelectedColumns()
   }
 
-  ngOnInit () {
+  async ngOnInit () {
     this.initialize()
 
-    this.bulkActions = [
+    this.pluginService.addAction('admin-user-list:load-data', this.reloadDataInternal.bind(this))
```

Comment:
So we don't accidentally break plugin API when we update `reloadDataInternal` parameters below, prefer to create a dedicated callback. For example

```ts
this.pluginService.addAction('admin-user-list:load-data', (options: { ... }) => this.reloadDataInternal({ ... }))
```

Reply:
Oh sorry I didn't see you could not pass params. But is is a `load-data` or a `reload-data` action then?

Reply:
When a plugin calls the action it will reload the data (reset). In one way `load-data` may be better to keep consistency if we'd like to add parameters to give plugins ability to pass query params to filter the data. On the other hand `reload-data` is more clearly for the plugin developer.

Reply:
I'll let you decide the implementation you want to develop. Both solutions are okay for me :)

Reply:
Lets keep load-data to be able to have it flexible.